server/views/pages/goal/update/review.njk

```diff
@@ -34,43 +34,44 @@
         {% endif %}
       {% endset %}
 
-      {{ govukSummaryList({
-        classes: 'govuk-!-margin-bottom-9',
-        rows: [
-          {
-            key: {
-              text: "Status"
-            },
-            value: {
-              html: goalStatusHtml
-            }
-          },
-          {
-            key: {
-              text: "Description"
-            },
-            value: {
-              text: data.title | formatMultilineText | safe
-            }
-          },
-          {
-            key: {
-              text: "When are they aiming to achieve this by?"
-            },
-            value: {
-              text: "by " + data.targetCompletionDate | formatDate('D MMMM YYYY')
-            }
-          },
-          {
-            key: {
-              text: "Note"
-            },
-            value: {
-              text: data.notes | formatMultilineText | safe
-            }
-          }
-        ]
-      }) }}
+      <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            Status
+          </dt>
+          <dd class="govuk-summary-list__value">
+            {{ goalStatusHtml | safe }}
+          </dd>
+        </div>
+
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            Description
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <span class="app-u-multiline-text">{{ data.title }}</span>
```

Comment:
To ensure the class is placed on the element immediately wrapping the text, and to ensure it's all on one line to prevent additional newline whitespace being rendered (see comment above) , I had to replace the use of the `govukSummaryList` with it's hand written html equivalent. 

There are 2 places in this component where I needed to do this, here on line 52, and the `notes` field on line 70